<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>765</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>635</width>
    <height>765</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>635</width>
    <height>765</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-image:url(:/res/img/Background.jpg)</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>130</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(255, 255, 255 ,0.6)
</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>420</y>
      <width>291</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

background:url(:/res/img/bak.jpg)rgb(200, 127, 160)
</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>231</width>
      <height>511</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(255, 255, 255 ,0.6)
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255,0.6)
</string>
    </property>
    <property name="text">
     <string>秋风清，秋月明，落叶聚还散，寒鸦栖复惊。相思相见知何日？此时此夜难为情!</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="NewTime">
    <property name="geometry">
     <rect>
      <x>-120</x>
      <y>0</y>
      <width>771</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/res/img/bak4.jpg)200 10 100 10;


</string>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionAdd_New"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
    <addaction name="actionUpdate"/>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>信息</string>
    </property>
    <addaction name="actionAppreciate"/>
    <addaction name="actionInfo"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionAdd_New"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionUpdate"/>
   <addaction name="actionSearch"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionAppreciate"/>
   <addaction name="actionInfo"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionAdd_New">
   <property name="icon">
    <iconset>
     <normalon>:/res/img/Add.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset>
     <normalon>:/res/img/Info.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>信息</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset>
     <normalon>:/res/img/Search.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </action>
  <action name="actionAppreciate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/res/img/Appreciate.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>赞赏</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/res/img/ReFresh.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normalon>:/res/img/Delete.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset>
     <normalon>:/res/img/UpDate.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>更改</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
